app/client/cli/doc/commands/client_Application_Stake.md

```diff
@@ -1,10 +1,18 @@
 ## client Application Stake
 
-Stake an actor (Application) in the network.
+Stake a node in the network. Custodial stake uses the same address as operator/output for rewards/return of staked funds.
 
 ### Synopsis
 
-Stake the Application actor into the network, making it available for service.
+Stake the node into the network, making it available for service.
```

Comment:
Optional nit: this formats as a single line in markdown github. Was that the intention or did you mean to split it out?

![Screen Shot 2022-11-09 at 11 03 19 AM](https://user-images.githubusercontent.com/1892194/200918390-fad84f4a-2788-4ed3-8d0b-35a2c4ac8c22.png)

I'd add new lines to make reading easier but doesn't really matter.

Reply:
Makes sense, updated 👍